contracts/Clearable.sol

```diff
@@ -0,0 +1,159 @@
+pragma solidity ^0.5.0;
+
+import "eip1996/contracts/Holdable.sol";
+import "./IClearable.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract Clearable is Holdable, IClearable, Ownable {
+
+    using StringUtil for string;
+
+    struct ClearableTransfer {
+        address orderer;
+        address from;
+        address to;
+        address clearableAgent;
+        uint256 value;
+        ClearableTransferStatusCode status;
+    }
+
+    mapping(bytes32 => ClearableTransfer) internal clearableTransfers;
+    mapping(address => mapping(address => bool)) private operators;
+    address clearableAgent;
```

Comment:
I suggest to call it clearingagent. instead clearableagent. It supposed to be an array but it makes the impl more complex due the notary will be an "open" array of clearing agent, or one well known agent